TestVectorsAwsCryptographicMaterialProviders/runtimes/python/fuzz_generator.py

```diff
@@ -0,0 +1,262 @@
+    
+    This strategy creates encryption contexts with:
+    - Normal ASCII strings
+    - Unicode characters (including the problematic ones from the UTF-8 vulnerability)
+    - Emojis and special characters
+    - Null bytes and control characters
+    - Surrogate pairs and replacement characters
+    
+    IMPORTANT: No empty strings for keys or values (invalid for KMS)
+    """
+    # Generate 1-20 key-value pairs (minimum 1 to ensure context exists)
+    num_pairs = draw(st.integers(min_value=1, max_value=20))
+    
+    context = {}
+    for _ in range(num_pairs):
+        # Generate diverse key names (minimum size 1 to avoid empty strings)
+        key = draw(st.one_of(
+            st.text(min_size=1, max_size=50),  # Normal text
+            st.text(min_size=1, max_size=50, alphabet=st.characters(whitelist_categories=['So', 'Sc', 'Sk'])),  # Symbols
+            st.text(min_size=1, max_size=50, alphabet=st.characters(whitelist_categories=['Lo', 'Ll', 'Lu'])),  # Letters
+            st.text(min_size=1, max_size=50, alphabet=st.characters(whitelist_categories=['Nd', 'Nl', 'No'])),  # Numbers
+            st.text(min_size=1, max_size=50, alphabet=st.characters(whitelist_categories=['Mn', 'Mc', 'Me'])),  # Marks
+            st.text(min_size=1, max_size=50, alphabet=st.characters(whitelist_categories=['Zs', 'Zl', 'Zp'])),  # Separators
+            st.text(min_size=1, max_size=50, alphabet=st.characters(whitelist_categories=['Cc', 'Cf', 'Cs', 'Co', 'Cn'])),  # Control chars
+        ))
+        
+        # Generate diverse values (minimum size 1 to avoid empty strings)
+        value = draw(st.one_of(
+            st.text(min_size=1, max_size=100),  # Normal text
+            st.text(min_size=1, max_size=100, alphabet=st.characters(whitelist_categories=['So', 'Sc', 'Sk'])),  # Symbols
+            st.text(min_size=1, max_size=100, alphabet=st.characters(whitelist_categories=['Lo', 'Ll', 'Lu'])),  # Letters
+            st.text(min_size=1, max_size=100, alphabet=st.characters(whitelist_categories=['Nd', 'Nl', 'No'])),  # Numbers
+            st.text(min_size=1, max_size=100, alphabet=st.characters(whitelist_categories=['Mn', 'Mc', 'Me'])),  # Marks
+            st.text(min_size=1, max_size=100, alphabet=st.characters(whitelist_categories=['Zs', 'Zl', 'Zp'])),  # Separators
+            st.text(min_size=1, max_size=100, alphabet=st.characters(whitelist_categories=['Cc', 'Cf', 'Cs', 'Co', 'Cn'])),  # Control chars
+            st.text(min_size=1, max_size=100, alphabet=st.characters(whitelist_categories=['So'])),  # Emojis and symbols
+        ))
+        
+        context[key] = value
+    
+    return context
+
+
+@composite
+def fuzz_test_vector(draw):
+    """
+    Generate a complete fuzzed test vector.
+    
+    This creates test vectors with:
+    - Fuzzed encryption contexts (HIGH PRIORITY) - UTF-8 fuzzing
+    - Normal ASCII for descriptions and other fields
+    - Focus on cross-runtime UTF-8 vulnerability detection
```

Comment:
I think the focus of this file is only generating JSON not running the encrypt and decrypt which will actually test for cross runtime vulnerablity.

Reply:
Why only UTF-8? why not utf-16?